modules/core/04-channel/types/channel.go

```diff
@@ -126,3 +126,23 @@ func (ic IdentifiedChannel) ValidateBasic() error {
 	channel := NewChannel(ic.State, ic.Ordering, ic.Counterparty, ic.ConnectionHops, ic.Version)
 	return channel.ValidateBasic()
 }
+
+// SubsetOf returns true if Order is a valid subset of the provided Order.
+func (o Order) SubsetOf(order Order) bool {
+	if o == order {
+		return true
+	}
+
+	switch o {
+	case ORDERED:
+		if order == UNORDERED {
+			return true
+		}
+	case UNORDERED:
+		if order == ORDERED {
+			return false
+		}
```

Comment:
Another equivalent though likely less explicit:

```suggestion
	case ORDERED:
            order == UNORDERED
	case UNORDERED:
            return false
```